### DISCUSSION TITLE
Beep-Boop-Beep

### DISCUSSION THREAD

**Discussion by 0xBLCKLPTN** [oreal-solutions/motionlink-cli#15] [General]
I will try to help develop this project as much as I can ;)

I may need to take an assignment from TODO, but if not, I can be contacted at _bl4ckl1pt0n@gmail.com_.

**Comment by bats64mgutsi** [comment]
@blcklptn thanks for making yourself available. I look forward to working with you.

Yes, do feel free to take on any issue. I am available if you need clarity on something 🙂
